packages/runtime/src/components/builtin/Carousel/register.ts

```diff
@@ -39,9 +37,9 @@ export function registerComponent(runtime: ReactRuntime) {
         id: ElementID(),
         images: Images({
           preset: [
-            { key: uuid(), props: {} },
-            { key: uuid(), props: {} },
-            { key: uuid(), props: {} },
+            { key: 'image-1', props: {} },
+            { key: 'image-2', props: {} },
+            { key: 'image-3', props: {} },
```

Comment:
With the [change above](https://github.com/makeswift/makeswift/pull/1138/files#r2323345140), the keys don't have to be globally unique, just unique within the carousel's images.